jcl/src/java.base/share/classes/com/ibm/tools/attach/target/CommonDirectory.java

```diff
@@ -282,7 +293,7 @@ public static int notifyVm(int numberOfTargets) {
 	 * @return 0 on success
 	 */
 	public static int cancelNotify(int numberOfTargets) {
```

Comment:
Similarly to notifyVm(), I don't think we need this old API to avoid changing the one caller.

Reply:
True.